orbit/cmd/orbit/orbit.go

```diff
@@ -1307,6 +1314,46 @@ func getFleetdComponentPaths(
 		log.Error().Err(err).Msg("update metadata before getting components")
 	}
 
+	// "root", "targets", or "snapshot" signatures have expired, thus
+	// we attempt to get local paths for the targets (updater.Get will fail
+	// because of the expired signatures).
+	if updater.SignaturesExpired() {
+		log.Error().Err(err).Msg("expired metadata, using local targets")
+
+		// Attempt to get local path of osqueryd.
+		osquerydPath, err = updater.ExecutableLocalPath("osqueryd")
```

Comment:
can we use a `const` for the fleetd components?  "osqueryd" "desktop"

Reply:
Please review carefully in case I made a mistake :)

[Use constants for fleetd TUF targets](https://github.com/fleetdm/fleet/pull/23102/commits/4cc0348ea4c5fa11ed8605b4d31cd0058a26e6dc)